GTHC/web
# Forgot password

## Description
I added an API endpoint to respond to user password reset requests. A reset token is sent to the user's on file email address via email from no-reply@gthc.io

## Motivation and Context
If someone forgets their password, they need a way to be reset it and access their account.

## How Has This Been Tested?

- Ask Anesu or Aman for the password to the email (or ask Anesu to send you the secrets.yml file)
- run 'yarn dev:start'
- in an API client like postman or ARC, make a POST request to 'http://localhost:5000/api/v1/user/forgot_password' with the body {user_email: ""}
- Check your email
- See email with password reset link (link to doesn't do anything yet)

## Screenshots (if appropriate):
<img width="654" alt="screen shot 2019-01-02 at 6 41 32 pm" src="https://user-images.githubusercontent.com/15840164/50619412-c4cf3600-0ebe-11e9-8a03-74312c41fc10.png">

## Types of changes


## Checklist:
